sprout/sprout/settings.py

```diff
@@ -158,6 +158,19 @@
     }
 }
 
+CLEANUP_RULES = [
+    # appliance name , max appliance lifetime
+    {'name': r'^test', 'lifetime': dict(hours=1)},
+    {'name': r'^(long-test|i-)', 'lifetime': dict(hours=6)},
+    {'name': r'^(external-test|s_tpl|s_appl)', 'lifetime': dict(hours=12)},
+    {'name': r'^jenkins-template', 'lifetime': dict(hours=4)},
```

Comment:
I'm thinking we make all of these  `r'^jenkins(-|_)'` - so we match dash or underscore and don't have to adjust in future

Reply:
above comment for jenkins-template, jenkins, sprout, dockerbot